features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/api.js

```diff
@@ -526,6 +545,29 @@ class API extends Resource {
         return promised_update;
     }
 
+    /**
+     * Update an api via PUT HTTP method, Need to give the updated API object as the argument.
+     * @param apiId {Object} Updated graphQL schema which needs to be updated
+     * @param graphQLSchema
+     * @deprecated
```

Comment:
Why do we have deprecated annotation here?

Reply:
The unwanted annotation  has been removed with 2a6fbdd3ac9ce54ca7bc94a45328a41826836a80